addon/components/nypr-account-forms/reset.js

```diff
@@ -18,6 +18,7 @@ const FLASH_MESSAGES = {
 export default Component.extend({
   layout,
   store: service(),
+  flashMessages: service(),
```

Comment:
I'm not sure how the later `this.get('flashMessages').add()` worked at all with this missing